src/js/components/DataFilter/__tests__/__snapshots__/DataFilter-test.tsx.snap

```diff
@@ -1000,17 +1000,17 @@ exports[`DataFilter noForm 1`] = `
             </div>
             <div
               class="c19"
-              style="flex: 3.8 0 0px; cursor: ew-resize;"
+              style="flex: 3.8000000000000003 0 0px; cursor: ew-resize;"
```

Comment:
Note to reviewers, the floating point here I think is something specific to jest. In the browser, this renders as 3.8:

<img width="1399" alt="Screen Shot 2023-12-06 at 2 04 26 PM" src="https://github.com/grommet/grommet/assets/12522275/4b14e8d2-20c8-46de-9a61-696978884913">